README.md

```diff
@@ -1,44 +1,67 @@
 # Jamulus Website
 
-This is the home of the [Jamulus website](https://jamulus.io). The goal of this page is to explain what Jamulus is and how to use it.
+This is the content repository for [jamulus.io](https://jamulus.io). Text appearing in the Jamulus software application itself is part of the [main code repository](https://github.com/jamulussoftware/jamulus). The web site is built with Jekyll and Fox-CSS.
 
-## Adding pictures
-Unfortunately GitHub doesn't enable picture upload on the normal md editor. Therefore you can attempt to open an issue, upload your image, copy the link to the page you want to integrate it into and then don't submit the issue. See this answer on stack overflow: https://stackoverflow.com/a/26601810
+### Please also note that discussions should be in English so as to be open to all. 
 
-## Contribute a translation
+## Making changes to the English version
 
-**Before translating, please have a look at the [CONTRIBUTING.md](CONTRIBUTING.md) file to get familiar with our translation process. Afterwards, check if there's already a discussion thread for your language here: https://github.com/jamulussoftware/jamulus/discussions and participate in it**
+(See the translation process later in this document if you want to get involved with that.)
 
-Translations are handled by the [Polyglot Jekyll Plug-in](https://github.com/untra/polyglot).
-If you want to translate a file, you must first know where it is located on this repo:
+Make a fork of the Jamulus repository (button at [top right here](https://github.com/jamulussoftware/jamuluswebsite)). In your fork, you can edit the website pages (or of course use git on the command line if you prefer). When you are done, create a pull request that we will review, translate and publish in the next release.
```

Comment:
In theory, you could also edit the page directly. GH will then create a fork for you.